packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -508,38 +508,38 @@ export class ContextualMenuBase extends BaseComponent<IContextualMenuProps, ICon
   }
 
   private _renderSectionItem(
-    item: IContextualMenuItem,
+    sectionItem: IContextualMenuItem,
     menuClassNames: IMenuItemClassNames,
     index: number,
     hasCheckmarks: boolean,
     hasIcons: boolean
   ) {
-    const section = item.sectionProps;
-    if (!section) {
+    const sectionProps = sectionItem.sectionProps;
+    if (!sectionProps) {
       return;
     }
 
     let headerItem;
-    if (section.title) {
+    if (sectionProps.title) {
       const headerContextualMenuItem: IContextualMenuItem = {
-        key: `section-${section.title}-title`,
+        key: `section-${sectionProps.title}-title`,
         itemType: ContextualMenuItemType.Header,
-        text: section.title
+        text: sectionProps.title
       };
       headerItem = this._renderHeaderMenuItem(headerContextualMenuItem, menuClassNames, index, hasCheckmarks, hasIcons);
     }
 
-    if (section.items && section.items.length > 0) {
+    if (sectionProps.items && sectionProps.items.length > 0) {
       return (
-        <li role="presentation" key={section.key}>
+        <li role="presentation" key={sectionItem.key}>
```

Comment:
I like the renaming 👍 . As I stated, `sectionProps` can take a `key` property, so we could also go with something like this to have a fallback for all cases and leave it on the developer to know that keys of items in a list should be unique.
```suggestion
        <li role="presentation" key={sectionItem.key || sectionProps.key || `section-${index}`}>
```

Reply:
ok, great. Since sectionProps.key is more specific, I will put it first, then have sectionItem.key be its fallback.